pkg/config/app/kuma-cp/config.go

```diff
@@ -459,6 +459,8 @@ type ExperimentalConfig struct {
 	// Enables sidecar containers in Kubernetes if supported by the Kubernetes
 	// environment.
 	SidecarContainers bool `json:"sidecarContainers" envconfig:"KUMA_EXPERIMENTAL_SIDECAR_CONTAINERS"`
+	// If true uses Delta xDS to deliver changes to sidecars.
+	DeltaXds bool `json:"deltaXds" envconfig:"KUMA_EXPERIMENTAL_DELTA_XDS"`
```

Comment:
Also one further question for the ENV variable.
In the future we'll get rid of `EXPERIMENTAL` right? 
At that time, do we support both `KUMA_EXPERIMENTAL_DELTA_XDS ` and `KUMA_DELTA_XDS` or just support `KUMA_DELTA_XDS` ?